packages/help-center/src/hooks/use-launchpad-checklist.tsx

```diff
@@ -45,9 +45,7 @@ export const useLaunchpadChecklist = ( siteSlug: string | null, siteIntent: stri
 	const key = [ 'launchpad-checklist', siteSlug ];
 	const queryResult = useQuery( key, () => fetchLaunchpadChecklist( siteSlug, siteIntent ), {
 		retry: 3,
-		initialData: {
-			checklist: [],
-		},
+		placeholderData: { checklist: [] },
```

Comment:
placeholderData doesn't overwrite cached values ( which we _do_ store between sessions in https://github.com/Automattic/wp-calypso/blob/trunk/client/state/query-client.ts#L12-L19 )